test/e2e/sync/sync.go

```diff
@@ -65,43 +65,43 @@ func runSyncCMD(args ...string) (int, string, string) {
 
 // Create Specifications yaml file
 func createSpecFile(spec []pkg.Spec) error {
-	klog.Infof("STEP: Creating Spec file")
+	klog.V(4).Info("STEP: Creating Spec file")
 	dir, err := os.MkdirTemp(".", "spec")
 	if err != nil {
-		klog.Errorf("ERROR: %v", err)
+		klog.Errorf("unable to create temporary directory, err: %v", err)
 		return err
 	}
 
 	file, err := os.CreateTemp(dir, "spec.*.yaml")
 	if err != nil {
-		klog.Errorf("ERROR: %v", err)
+		klog.Errorf("unable to create a temp file, err: %v", err)
 		return err
 	}
 	defer file.Close()
 
 	SpecFileName = file.Name()
 	specString, merr := yaml.Marshal(&spec)
 	if merr != nil {
-		klog.Errorf("ERROR: %v", merr)
+		klog.Errorf("marshal failed, err: %v", merr)
 		return merr
 	}
 
 	_, err = file.WriteString(string(specString))
 	if err != nil {
-		klog.Errorf("ERROR: %v", err)
+		klog.Errorf("error while writing in the file, err: %v", err)
 		return err
 	}
 
-	klog.Infoln(string(specString))
+	klog.V(3).Info(string(specString))
```

Comment:
Can add more context: 
```suggestion
	klog.V(3).Info("Specifications for e2e sync test", string(specString))
```